### ISSUE TITLE
Disclosure, marketing context

### ISSUE THREAD

**Issue by ajacobs21e** [patternfly/patternfly-design#874]
Requesting a disclosure component for the marketing context. 
https://www.sketch.com/s/f6a52c25-8657-4680-9d3e-4c09131e6ed0/p/disclosure

This is similar in function to the `expandable` component in PF4 but with more visual weight. Visually it is similar to a single accordion.

**Comment by mcarrano** [comment]
Would an accordion work here @ajacobs21e ?

**Comment by ajacobs21e** [comment]
@coreyvickery do you have more requirements on when a disclosure is needed instead of an accordion?

@mcarrano one difference is the location of the caret. Expandable and data list have the caret on the left and accordion has it on the right. In a full width example like the top one here, a user might not notice the caret all the way on the right
https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/e5f80536-7da1-421e-813a-1cb974cfcd02/Disclosure-light-theme

**Comment by coreyvickery** [comment]
@ajacobs21e I don't have any concrete requirements, but I've always thought that it's up to the designer to figure out based on the content they have to work with. The link below has an example of a Disclosure that displays cross-links on a redhat.com search page.

https://www.redhat.com/en/search/RHEL

In this case, the Disclosure is displaying cross-links to another website (Customer Portal). I believe Disclosures can be used to store content that's not crucial to the user experience, what's inside is supportive or extra. If someone is putting content that's too dense and can be organized into sections, an Accordion or even Tabs would be more appropriate.

I hope that makes sense.

**Comment by mcarrano** [comment]
Consider whether this can be achieved with an expandable card.

**Comment by coreyvickery** [comment]
@mcarrano Seems to function the same, but stylistically, we would need a border around that card so it kind of resembles an Accordion panel. See this issue that I wrote for more details.

https://github.com/patternfly/patternfly-elements/issues/829

**Comment by mcarrano** [comment]
@ajacobs21e @coreyvickery I'm taking a look again at this issue. Since we already have multiple things like this in the library, I'd rather not add a new component for this. I feel like the designs for this are pretty close to the expandable card that we added since this issue was opened. You can see that here: https://www.patternfly.org/v4/components/card/#expandable The only difference I see is the border styling on the expansion, which I think we could address.

Any reason that the expandable card component would not meet your needs?

**Comment by coreyvickery** [comment]
@mcarrano Are you saying that you want to restyle the Expandable card so that it matches an expanded Accordion panel? In marketing, we don't use a Disclosure that often, but we do use it, and it's nice that it pairs with the styling of an Accordion. Additionally, our Card component is very complex and has lots of rules, so I don't know if our developers would be happy adding more complexity to that.

**Comment by mcarrano** [comment]
@coreyvickery One of the reasons I suggested the expandable card was that in looking at your mocks, they seem to have a very similar structure to our cards with a header, a body, and a footer. But if you would prefer something that just provides a less structured container, I think that we can build this on top of the existing Exandable section component: https://www.patternfly.org/v4/components/expandable-section. It already has the behavior you are looking for. I would approach this by introducing a new variant that has more accordion like styling. Thoughts?

**Comment by coreyvickery** [comment]
@mcarrano So it sounds like we can't change the name after all or if we tried, it would break a lot of things. See Kendall's comment below. Can the marketing version of `Expandable section` simply be named `Disclosure`, but look and act the same?

https://github.com/patternfly/patternfly-elements/issues/829#issuecomment-789852893

cc @mceledonia @ajacobs21e

**Comment by mcarrano** [comment]
@coreyvickery Yeah, we have the same issue with renaming things in PatternFly. Not sure I understand your question exactly. Are you just asking that the example on the PatternFly website within the Expandable section component be named "Disclosure?" Sure we can do that. But it would still live as a variant of https://www.patternfly.org/v4/components/expandable-section

**Comment by coreyvickery** [comment]
@mcarrano Sort of, you don't have to change anything, I was just asking if we can keep the `Disclosure` name, but it will be styled and function like PatternFly's `Expandable section`.

**Comment by mcarrano** [comment]
Yes, I think we can.

**Comment by mcarrano** [comment]
A core issue is opened here to begin implementation: https://github.com/patternfly/patternfly/issues/3952
